reference_guide/intellij_artifacts.md

```diff
@@ -25,27 +25,58 @@ More information about [Gradle support](https://www.jetbrains.com/help/idea/grad
 
 To setup dependencies on a module there are two types of information needed:
 1. Specify the corresponding repository URL for the artifact.
-2. Specify the Maven coordinates for the artifact. 
+2. Specify the [Maven coordinates](https://maven.apache.org/pom.html#Maven_Coordinates) for the artifact. 
  
 ### Specify the Repository URL 
 The URL for the desired artifact needs to be added to a Maven or Gradle script:
 * For release or EAP versions, use `https://www.jetbrains.com/intellij-repository/releases` 
 * For EAP candidate snapshots, use `https://www.jetbrains.com/intellij-repository/snapshots`
 * For dependencies on individual modules from the IntelliJ Platform, also use `https://jetbrains.bintray.com/intellij-third-party-dependencies` 
 
-### Specify the Artifact
-Describing a desired IntelliJ Platform module artifact is done with Maven coordinates. 
+### Specify the Maven Coordinates for the Artifact
+Describing a desired IntelliJ Platform module artifact is done with Maven coordinates: _groupId_, _artifactId_, and _version_.
+The Maven coordinates are based on the names of modules.
+
+The _groupId_ for a module is the prefix `com.jetbrains.` concatenated with the first two parts of the module name.  
+For example, the module `intellij.xml` would have the groupId `com.jetbrains.intellij.xml`. 
+
+The _artifactId_ is the second.._n_ parts of the module name separated by "-" characters. 
+For example, the module `intellij.xml` would have the artifactId `xml`.
+There are some special cases to artifactId names. 
+If the second part of the module name is a common group like `platform`, `vcs`, or `cloud`, the second part of the module name is dropped, and the artifactId becomes the third.._n_ parts of the module name, separated by "-" characters.
+Portions of the module name expressed in `camelCase` format are divided and used in the artifactId as (all lower case) `camel-case`.  
+
+The table below shows some module names and their corresponding groupId and artifactId.
+
+| **Module Name** | **groupId** | **artifactId** | **Module Name Length, Comments About artifactId** |
+|-----------------|-------------|----------------|-------------------------------|
+|intellij.xml | com.jetbrains.intellij.xml | xml | n=2 |
```

Comment:
I think it would be better to sort the rows, it looks strange when java and platform modules are mixed together. I would either sort them alphabetically, or move platform modules to the top.

Reply:
Sorted rows alphabetically by module name.